src/ApparentHorizons/YlmSpherepack.cpp

```diff
@@ -0,0 +1,884 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cmath>
+#include <iostream>
```

Comment:
No iostream, don't we have a git hook which checks for this?

Reply:
No, we have a Travis check, since for debugging purposes committing it might be desired.

Reply:
Yeah, I put iostream in when debugging and forgot to remove it.